src/definitions/v.json

```diff
@@ -0,0 +1,114 @@
+[
+    {
+      "display": "Vertical Align",
+      "description": "Sets the vertical alignment of an element",
+      "link": "https://www.w3schools.com/cssref/pr_pos_vertical-align.asp",
+      "propertyName": "vertical-align",
+      "valuetemplate": "{0}",
+      "categories": ["alignment", "layout", "position"],
+      "tags": "alignment, layout, position",
+      "controls": [
+        {
+          "type": "select",
+          "default": "baseline",
+          "label": "Value",
+          "options": [
+            {
+                "name": "Align to baseline of container",
+                "value": "baseline"
+            },
+            {
+                "name": "Custom",
+                "valuetemplate": "{0}px",
+                "value": {
+                  "controls": [
+                    {
+                      "type": "number",
+                      "label": "Length"
+                    }
+                  ]
+                }
+            },
+            {
+                "name": "Percentage",
```

Comment:
We would need to accompany % with some custom value like "px". So, you can create one `Custom` option which will render two-element. One `number box` to enter the value and one `select` to assign `%` or `px`.
Refer `border-radius`

Reply:
Fixed!  Updated `Length (Custom)` section.  Thank you for the review!